extensions/parquet/base/src/main/java/io/deephaven/parquet/base/PlainBinaryChunkedWriter.java

```diff
@@ -9,21 +9,17 @@
 import org.apache.parquet.column.Encoding;
 import org.apache.parquet.column.values.rle.RunLengthBitPackingHybridEncoder;
 import org.apache.parquet.io.api.Binary;
-import org.slf4j.Logger;
```

Comment:
We have no replicate for this file - interestingly, dhe _does_, but has no autogenerated header. I'm going to see if we can get the file to actually be replicated, but in the meantime since dhe has a replicate tool and had this change (unused logger, unused null count) it seemed to make sense to prune.

Reply:
Pruning is fine. Pruning the false header comment should happen, too. I suspect pretty strongly that they (DHE) have not gone in and written a replicator for this file. We have one that works (with supervision) for the numeric types, not the "special" types (see https://github.com/deephaven/deephaven-core/pull/2376#discussion_r869765838).

Reply:
The version of DHE code I'm looking at does not have replication for this, Boolean, or FixedLen in `io.deephaven.replicate.parquet.Replicate`.

Reply:
Sorry, I misunderstood initially - you are commenting on this line, but you actually want the header to be removed for 3 files (plainbinary, boolean, and fixedlen), right?